website/docs/r/agent_pool.html.markdown

```diff
@@ -37,6 +38,9 @@ The following arguments are supported:
 
 * `name` - (Required) Name of the agent pool.
 * `organization` - (Optional) Name of the organization. If omitted, organization must be defined in the provider config.
+* `organization_scoped` - (Optional) Whether or not the agent pool is scoped to all workspaces in the organization. Defaults to `true`.
+* `allowed_workspace_ids` - (Optional) IDs of the workspaces that to which agent pools is scoped.
+    If `organization_scoped` is set to `true`, this argument will not be served and agent pool will be scoped to all the workspaces.
```

Comment:
`...this argument will not be served...`, this part of statement seems unclear to me, does this mean the `allowed_workspace_ids` attribute will be absent or set to null or sent to an empty slice?

Reply:
Removed

Reply:
Moved `allowed_workspace_ids` to r/agent_pool_allowed_workspaces resource.